pydm/utilities/connection.py

```diff
@@ -0,0 +1,55 @@
+from qtpy.QtWidgets import QWidget
+
+
+def __change_connection_status(widget, status):
```

Comment:
General comment: should consider using double underscore sparingly. A single underscore is usually sufficient, with the double underscore making poking around and (if ever necessary) monkey-patching rather frustrating.